_search-plugins/sql/functions.md

```diff
@@ -33,20 +33,24 @@ ln | `ln(number T) -> double` | `SELECT ln(10) FROM my-index LIMIT 1`
 log | `log(number T) -> double` or `log(number T, number) -> double` | `SELECT log(10) FROM my-index LIMIT 1`
 log2 | `log2(number T) -> double` | `SELECT log2(10) FROM my-index LIMIT 1`
 log10 | `log10(number T) -> double` | `SELECT log10(10) FROM my-index LIMIT 1`
-mod | `mod(number T, number) -> T` | `SELECT modulus(2, 3) FROM my-index LIMIT 1`
-multiply | `multiply(number T, number) -> number` | `SELECT multiply(2, 3) FROM my-index LIMIT 1`
+mod | `mod(number T, number T) -> T` | `SELECT mod(2, 3) FROM my-index LIMIT 1`
+modulus | `modulus(number T, number T) -> T` | `SELECT modulus(2, 3) FROM my-index LIMIT 1`
+multiply | `multiply(number T, number T) -> T` | `SELECT multiply(2, 3) FROM my-index LIMIT 1`
 pi | `pi() -> double` | `SELECT pi() FROM my-index LIMIT 1`
-pow | `pow(number T) -> T` or `pow(number T, number) -> T` | `SELECT pow(2, 3) FROM my-index LIMIT 1`
-power | `power(number T) -> T` or `power(number T, number) -> T` | `SELECT power(2, 3) FROM my-index LIMIT 1`
+pow | `pow(number T, number T) -> T` | `SELECT pow(2, 3) FROM my-index LIMIT 1`
+power | `power(number T, number) -> T` | `SELECT power(2, 3) FROM my-index LIMIT 1`
 rand | `rand() -> number` or `rand(number T) -> T` | `SELECT rand(0.5) FROM my-index LIMIT 1`
 rint | `rint(number T) -> T` | `SELECT rint(1.5) FROM my-index LIMIT 1`
 round | `round(number T) -> T` | `SELECT round(1.5) FROM my-index LIMIT 1`
 sign | `sign(number T) -> T` | `SELECT sign(1.5) FROM my-index LIMIT 1`
 signum | `signum(number T) -> T` | `SELECT signum(0.5) FROM my-index LIMIT 1`
 sqrt | `sqrt(number T) -> T` | `SELECT sqrt(0.5) FROM my-index LIMIT 1`
 strcmp | `strcmp(string T, string T) -> T` | `SELECT strcmp('hello', 'hello') FROM my-index LIMIT 1`
-subtract | `subtract(number T, number) -> T` | `SELECT subtract(3, 2) FROM my-index LIMIT 1`
+subtract | `subtract(number T, number T) -> T` | `SELECT subtract(3, 2) FROM my-index LIMIT 1`
 truncate | `truncate(number T, number T) -> T` | `SELECT truncate(56.78, 1) FROM my-index LIMIT 1`
++ | `number [op] number -> number` | `SELECT 1 + 100 FROM my-index LIMIT 1`
```

Comment:
Shouldn't we replace `[op]` with exact operation since they are listed one per line?

Reply:
That's a good point, I was just copying the syntax that was already there but putting the operation in makes more sense.